spaces/S000055/properties/P000136.md

```diff
@@ -0,0 +1,12 @@
+---
+space: S000055
+property: P000136
+value: true
+refs:
+  - mathse: 4580731
+    name: Is there a connected T1 anticompact topological space?
+  - doi: 10.1007/978-1-4612-6290-9 
+    name: Counterexamples in Topology
+---
+
+Item 7 of space 63 in {{doi:10.1007/978-1-4612-6290-9}}, see also {{mathse:4580731}}.
```

Comment:
Really nitpicking here, but for consistency with other spaces should we write "item #7 in space #63" ?

Reply:
Sure, I'll change that when I'm back in the office next week.